control_toolbox/include/control_toolbox/pid.hpp

```diff
@@ -426,15 +644,14 @@ class Pid
   }
 
 protected:
-  // Store the PID gains in a realtime buffer to allow dynamic reconfigure to update it without
-  // blocking the realtime update loop
   realtime_tools::RealtimeBuffer<Gains> gains_buffer_;
```

Comment:
this comment still applies?
```suggestion
  // Store the PID gains in a realtime buffer to allow dynamic reconfigure to update it without
  // blocking the realtime update loop
  realtime_tools::RealtimeBuffer<Gains> gains_buffer_;
```

Reply:
I've deleted it at first because I thought dynamic reconfigure is ROS1 related. Added in commit d9a6f2e.

Reply:
yes, the dynamic_reconfigure was a package in ROS 1 to specifically enable this, but the term describes it still well for ROS 2 ;)